io.sloeber.feature/feature.xml

```diff
@@ -32,6 +32,11 @@ Trump211
 If you do not agree with this policy, please do not install this software.
    </license>
 
+   <url>
+      <discovery label="Eclipse Nebula Widgets" url="https://download.eclipse.org/nebula/releases/latest/"/>
+      <discovery label="Eclipse Nebula Incubation Widgets" url="https://download.eclipse.org/nebula/incubation/latest"/>
```

Comment:
Do we still want to add this?
This has been added after you did some changes to the plotter Sloeber needed but the plotter has been released and stable for a long time now.

Reply:
Yes. When do not install from the product this will add the required nebula repos.

Reply:
I also updated the target file here to include 0.0.0 versions

Reply:
Thanks.
It was on my todo list.